src/VisualStudio/Core/Def/TableDataSource/Suppression/VisualStudioDiagnosticListSuppressionStateService.cs

```diff
@@ -299,13 +299,12 @@ public async Task<ImmutableArray<DiagnosticData>> GetSelectedItemsAsync(bool isA
                             continue;
                         }
 
-                        var tree = await document.GetRequiredSyntaxTreeAsync(cancellationToken).ConfigureAwait(false);
+                        var text = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
                         var linePosition = new LinePosition(line, 0);
                         var linePositionSpan = new LinePositionSpan(start: linePosition, end: linePosition);
-                        var textSpan = (await tree.GetTextAsync(cancellationToken).ConfigureAwait(false)).Lines.GetTextSpan(linePositionSpan);
-                        var location = new DiagnosticDataLocation(filePath, document.Id, textSpan,
-                            originalStartLine: linePosition.Line, originalStartColumn: linePosition.Character,
-                            originalEndLine: linePosition.Line, originalEndColumn: linePosition.Character);
+                        var textSpan = text.Lines.GetTextSpan(linePositionSpan);
+                        var location = new DiagnosticDataLocation(
+                            new FileLinePositionSpan(filePath, linePositionSpan), document.Id, textSpan, mappedFileSpan: null);
```

Comment:
note: i'm preserving semantics.  but i'm not at all sure if this is what should be done here.  namely, once we have the text span for the linepositionspan, it would probably make sense to actually go to the tree and see if that actually is mapped somewhere.